README.md

```diff
@@ -1,6 +1,201 @@
 Job Restrictions Plugin for [Jenkins CI][1]
 =======================
-See [Job Restrictions Plugin Page][2] for more information.
+
+| Plugin Information |
+|---------------------------------------------------------------------------------------------------------------|
+| View Job Restrictions [on the plugin site](https://plugins.jenkins.io/job-restrictions) for more information. |
+
+**This plugin is up for adoption.** Want to help improve this plugin?
+[Click here to learn
+more](https://wiki.jenkins.io/display/JENKINS/Adopt+a+Plugin "Adopt a Plugin")!
+
+The plugin allows restricting job executions in order to change their
+behavior or to harden the security. With this plugin it is possible to
+configure nodes to accept only particular jobs. It is also possible to
+prevent job triggering by users and other jobs using various conditions.
+
+# About the plugin
+
+### Features:
+
+-   Execution restrictions on the node level
+    -   Example 1: Take jobs according to the specified name pattern
+        (e.g. allow running only "QA\_.\*" jobs on a node)
+    -   Example 2: Prevent execution of user jobs on the Jenkins master
+        node
+-   Restrictions of jobs triggering by external causes
+    -   Example 1: Prohibit manual builds
+    -   Example 2: Allow triggering only by a job with the
+        specified [owner](https://wiki.jenkins.io/display/JENKINS/Ownership+Plugin).
+-   Several built-in restrictions + JobRestriction extension point
+
+### Contributors:
+
+1.  [Oleg Nenashev](hhttp://www.linkedin.com/in/onenashev) (maintainer)
+    -   Initial plugin version has been sponsored by Synopsys Inc.,
+        [www.synopsys.com](http://www.synopsys.com/)
+
+# Usage
+
+## Job restrictions for nodes
+
+These restrictions can be configured in node's configuration page.
+
+-   Node won't accept jobs, which violate the specified restrictions
+-   Other requirements (executors, labels, etc.) will be checked as well
+
+![](https://wiki.jenkins.io/download/attachments/69273423/Jenkins_JobRestrictions_NodeRestriction.png?version=1&modificationDate=1378665445000&api=v2)
```

Comment:
Assuming you used the wiki exporter, can you use the zip format so it gives you the attachments as well? this way the docs won't still refer to the wiki

Reply:
@halkeye Pardon my ignorance, but what is the best practice for linking from a Zip file in Github-flavored Markdown? Thanks again for the feedback.

Reply:
use https://jenkins-wiki-exporter.jenkins.io/ to export a zip
then extract the zip into a directory, and commit that.
@MarkEWaite I think has better tutorials

Reply:
See the [YouTube video tutorial](https://www.youtube.com/watch?v=BaEJ8v7INNQ&list=PLN7ajX_VdyaNll_Aw3M-NIQ042BV5teP9&index=6) or the video on [user documentation migration](https://www.youtube.com/watch?v=KB-NPlRvLoY&list=PLN7ajX_VdyaNll_Aw3M-NIQ042BV5teP9&index=7).

Reply:
Thanks I think I figured it out. Added to `docs/images/` directory and then linked from there within the README file. Thanks. I think I'm ready for re-review.